app/javascript/gobierto_dashboards/webapp/components/WidgetIndicator.vue

```diff
@@ -39,7 +39,7 @@
               :value="key"
               required
             >
-            <span>{{ name }}</span>
+            <span>{{ name() }}</span>
```

Comment:
Hmmmm I've seen this code in another PR? Or maybe I already reviewed this PR....

Reply:
Yeah, it sounds odd. I've merged `master` so this change (which I think it belongs to the previous PR https://github.com/PopulateTools/gobierto/pull/3581/files) shouldn't be here.

Reply:
I merged again master, now there are only two changed files.